### ISSUE TITLE
More/Less Hyperlink Breaks on Functionality/List Modules

### ISSUE THREAD

**Issue by prevoj** [backdrop-contrib/eu_cookie_compliance#14]
I have two Backdrop CMS 1.22.0 sites where the more/less hyperlink on Functionality/List Modules wouldn't work. After a lot of trial and error, I determined that as soon as I disabled eu_cookie_compliance, the more/less hyperlink started working again. I went to a completely different site that was working fine, installed and enabled this module and the more/less hyperlink immediately stopped working.

After the List Modules screen comes up, I can manually run Backdrop.behaviors.moduleFilter.attach() in the Chrome console and restore the more/less functionality temporarily until the page reloads. So I'm thinking that this modules is somehow preventing that function from running or causing it to run at the wrong time.

Joe Prevo

**Comment by argiepiano** [comment]
I can reproduce this problem. I did a bit of debugging, and noticed that the click listener in line 19 of `modules.js` is being attached twice to the 'more/less' DOM elements. This causes the callback to be called twice, effectively opening and closing the toggle (it's visually impossible to see it open and close - you can only see that with a js breakpoint).

It's unclear to me why EU Cookie Compliance is causing this double-attachment, but I'm wondering if perhaps this issue may also be present with other jquery listeners throughout the Backdrop system when this module is installed?

**Comment by argiepiano** [comment]
As suspected, other element listeners are also affected. For example, the more/less toggles in the Status Report (admin/reports/status)

**Comment by laryn** [comment]
I just came across this on a site as well, and eventually found my way here while trying to figure out why the more/less links weren't working.

**Comment by swampopus** [comment]
This issue is still going on for the latest version. The easiest place to see it is on the admin/reports/status screen.  All of the "more" and "less" links get "clicked" twice when you click them.  The code in core is dead simple, so it must be this module.  The only workaround I can see is under "advanced" to just exclude every path where this happens.

**Comment by swampopus** [comment]
I was able to fix this with the following pull request:
https://github.com/backdrop-contrib/eu_cookie_compliance/pull/25

**Comment by argiepiano** [comment]
This approach seems to be correct - there shouldn't be a need to invoke `Backdrop.attachBehaviors()` manually here. The D7 version of the module doesn't do it. 

@swampopus, would you please remove the commented code in your PR? 

There seem to be several problematic issues reported for this module. Pinging @Graham-72  (maintainer) to see if this fix can be merged and a new version of the module can be released. This issue seems serious enough for the @backdrop-contrib/bug-squad to merge, but we can't make new releases unless the module is abandoned.

**Comment by swampopus** [comment]
@argiepiano  No problem.  I might have done this the wrong way (somewhat new to github I'm sorry to say, even though I've been a professional dev for almost 30 years :/), but I just closed my previous pull request and after committing my changes, I added a new one:
https://github.com/backdrop-contrib/eu_cookie_compliance/pull/26

The commented-out line of code was removed.

**Comment by swampopus** [comment]
Also-- I like that the bug-squad exists-- I didn't even know that was a thing.  Do we just tag them like you did to get their attention on these sorts of things?  I've dealt with several modules now that I've provided PRs or issues for, but the module hasn't been worked on in 8+ years, so I doubt the original maintainer is going to do anything with it.

It's a shame, too-- I imagine _now_ is when we'd see a lot of D7 devs moving to Backdrop, since the D7 EOL just happened.  But some of these modules are **really old**, and some even have compatibility issues with PHP 8.  Nothing is going to hurt the effort to convert a D7 dev to Backdrop like buggy modules, modules only on github (no official releases) and, worst of all, warnings or errors on trying to install the module (which I've found a few times).

**Comment by argiepiano** [comment]
Yes, the [bug-squad](https://backdropcms.org/leadership/bug-squad) is a thing. It's ok to ping the squad. See the criteria for new releases in that page (most of them apply to this case - pinging @herbdool, who's part of the squad, to see if he agrees to do a release.

**Comment by argiepiano** [comment]
BTW there wasn't a need to do a new PR. You can just push the changes to the existing one.

**Comment by argiepiano** [comment]
I've merged this on behalf of the @backdrop-contrib/bug-squad

**Comment by herbdool** [comment]
I agree with @argiepiano. I've merged and created a new release

**Comment by swampopus** [comment]
Very cool, thanks for the tip, and thanks for the new release
